ext/pdo_mysql/tests/gh20122.phpt

```diff
@@ -8,11 +8,10 @@ pdo_mysql
 require_once __DIR__ . '/inc/mysql_pdo_test.inc';
 MySQLPDOTest::skip();
 ?>
---XFAIL--
 --FILE--
 <?php
 require_once __DIR__ . '/inc/mysql_pdo_test.inc';
-$pdo = MySQLPDOTest::factory();
+$db = MySQLPDOTest::factory();
 
 $db->exec('CREATE TABLE test (bar JSON)');
 $db->exec('INSERT INTO test VALUES("[]")');
```

Comment:
Would also be nice to make the tests parallelable by not using `test` as the table name